docs/editors/sublime.md

```diff
@@ -37,31 +37,19 @@ definition.
 ]
 ```
 
-## Importing a build
 
-Open Sublime in the base directory of an sbt build. Run the "Enable Language
-Server in project" command.
+## Importing a build
 
-![Enable Language Server for this project](https://i.imgur.com/3c0ZSZm.gif)
+Open Sublime in the base directory of an sbt build and it will then prompt you to import the build. Click "Import build" to start the installation step.
```

Comment:
```suggestion
Open Sublime in the base directory of your Scala project and it will then prompt you to import the build as long as you're using one of the [supported build tools](https://scalameta.org/metals/docs/build-tools/overview.html). Click "Import build" to start the installation step.
```
What do you think about not being so specific to sbt? I just looked and we phrased it this way in the vscode docs as well, but I sort of feel like this is more accurate. What do you think?